app/uk/gov/hmrc/preferenceschangednotifier/controllers/model/RestInstantFormat.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.preferenceschangednotifier.controllers.model
+
+import play.api.libs.json.{
+  Format,
+  JsSuccess,
+  JsError,
+  JsResult,
+  JsValue,
+  JsString,
+  Reads,
+  Writes
+}
+
+import java.time.Instant
+import scala.util.Try
+
+object RestInstantFormat {
+  implicit val format = Format(restInstantReads, restInstantWrites)
+
+  private lazy val restInstantReads: Reads[Instant] =
+    new Reads[Instant] {
+      override def reads(json: JsValue): JsResult[Instant] = {
+        json match {
+          case JsString(s) =>
+            Try(Instant.parse(s))
+              .fold(
+                _ => JsError(s"Could not parse $s as an ISO Instant"),
+                JsSuccess.apply(_)
+              )
+          case _ =>
+            JsError(s"Expected value to be a string, was actually $json")
+        }
+      }
+    }
+
+  private lazy val restInstantWrites: Writes[Instant] =
+    new Writes[Instant] {
+      private val restDateTimeFormat =
```

Comment:
just a minor comment really, you could disagree - shouldn't this be named as `restInstantFormat` to avoid the confusion, we have only **Instant** data type and there is no reference to **DateTime** data type?

Reply:
Hi @kjainsc , thanks for catching this, have rectified as part of the refactor for Muralis comments above.